app/build.gradle

```diff
@@ -26,6 +26,9 @@ android {
     }
```

Comment:
**Reporter**: ANDROIDLINT
**Rule**: Correctness
**Severity**: WARN
**File**: app/build.gradle L40

GradleDependency: Obsolete Gradle Dependency
A newer version of com.android.support:recyclerview-v7 than 25.1.0 is available: 25.3.1
This detector looks for usages of libraries where the version you are using is not the current stable release. Using older versions is fine, and there are cases where you deliberately want to stick with an older version. However, you may simply not be aware that a more recent version is available, and that is what this lint check helps find.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>*